modules/web-console/web-console-server/pom.xml

```diff
@@ -36,6 +36,7 @@
     <properties>
         <commons.compress.version>1.18</commons.compress.version>
         <spring.boot.version>1.5.20.RELEASE</spring.boot.version>
+        <swagger2.version>2.9.2</swagger2.version>
```

Comment:
Can this properties be moved to `parent/pom.xml` as well?

Reply:
Moved